e2e/multisig-migration.test.ts

```diff
@@ -52,6 +50,17 @@ describe("Should change LBC owner to the multisig", function () {
     ).to.not.be.reverted;
     const newLbcOwner = await lbc.owner();
     console.info("New LBC owner:", newLbcOwner);
+
+    await expect(
+      lbc.connect(impersonatedSigner).setProviderStatus(1, false)
+    ).to.be.revertedWith("LBC005");
+
+    expect(
```

Comment:
```suggestion
    await expect(
```

Reply:
This is using the eventually, so there is no need of await.